Dashboard docs (#433)
returntocorp/semgrep-docs: docs/semgrep-app/dashboard.md

| Orange | There are some impactful vulnerabilities detected for this category relative to other categories. |
| Red | There are plenty of impactful vulnerabilities for this category relative to other categories. Take action to reduce them by fixing the issues or identifying false positives. |

## Assessing rule performance

![Screenshot of dashboard rules widget](../img/dashboard-rules.png)

The **Rules widget** provides a summary report for rule metrics, such as what rules are ignored or fired the most.

These data points can serve as a starting point for the following security audits:

- Investigating the relevance or quality of a rule — is it useful, or does it detect too many false positives?
- Are there underlying issues in the codebase that result in recurring patterns of insecure code?
- Are there rules that developers don’t resolve? Semgrep helps identify these, which helps to form insights into possible causes.

### Rule performance through Developer Feedback

![Screenshot of developer feedback](../img/dashboard-devfeedback.png)

In Team/Enterprise Dashboards, these additional metrics are visible:

* Most disliked
* Slowest

> The **Most disliked** metric is taken from the **Developer Feedback** feature. It is a simple poll conducted by the Semgrep bot within Github in which developers vote on the usefulness of a rule. This feature is available for Team/Enterprise users.

Reviewer: ```suggestion
The **Most disliked** metric is taken from the **Developer Feedback** feature. Developers are able to provide feedback on findings that are not the most accurate. This feature is available for Team/Enterprise users.
```
